egui-wgpu/src/renderer.rs

```diff
@@ -38,6 +38,7 @@ impl ScreenDescriptor {
 #[repr(C)]
 struct UniformBuffer {
     screen_size_in_points: [f32; 2],
+    _padding: [u32; 2],
```

Comment:
You can write your findings, that you describe in ticket, here as doc comment for further readers, because otherwise intentions of this field a very unclear!

Reply:
100%

```suggestion
    _padding: [u32; 2], // Required on Chromium, see https://github.com/emilk/egui/pull/1671
```